launch_ros/launch_ros/actions/load_composable_nodes.py

```diff
@@ -133,28 +133,25 @@ def _load_node(
                 )
             ]
         response = self.__rclpy_load_node_client.call(request)
+        node_name = response.full_node_name if response.full_node_name else request.node_name
```

Comment:
why do we need to check that `response.full_node_name` is not empty? is that happening?

Reply:
Not sure, but this is also carry-over from existing logic at https://github.com/ros2/launch_ros/blob/5f54cb27d1038a2ec52d3a577a60791be1df1a47/launch_ros/launch_ros/actions/load_composable_nodes.py#L138

Reply:
sounds good